pkgs/development/python-modules/vqgan-jax/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, flax
+, jaxlib
+, pytorch
+, transformers
+}:
+
+buildPythonPackage rec {
+  pname = "vqgan-jax";
+  version = "unstable-2022-04-20";
+
```

Comment:
```suggestion

  format = "setuptools";

```

Reply:
This is the default

Reply:
It might change in the future, that's why all packages should specify it.